answerking_app/tests/test_orders.py

```diff
@@ -474,20 +498,22 @@ def test_put_invalid_address_returns_bad_request(self):
         actual = response.json()
 
         # Assert
-        self.assertEqual(expected_failure_error, actual)
+        self.assertIsInstance(actual.pop("traceId"), str)
+        self.assertEqual(expected, actual)
         self.assertEqual(response.status_code, 400)
 
-    def test_put_invalid_status_returns_bad_request(self):
+    def test_put_invalid_status_returns_not_found(self):
         # Arrange
-        invalid_post_data: UpdateOrderType = {
+        invalid_post_data: OrderType = {
             "address": "test",
             "status": "invalid",
         }
-        expected_failure_error: ErrorMessage = {
-            "error": {
-                "message": "Request failed",
-                "details": "Object could not be updated",
-            }
+        expected: DetailError = {
+            "detail": "Object was not Found",
+            "errors": ["Status matching query does not exist."],
```

Comment:
don't quite follow this error? would it not be better to put "invalid status given"?

Reply:
ok, I'll leave it for the moment, can be deleted in other branches